dev/com.ibm.ws.ejbcontainer.timer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -36,3 +36,9 @@ persistentExecutorRef.desc=Schedules and runs EJB persistent timer tasks.
 nonPersistentContextServiceRef=Non-persistent Timer Thread Context Propagation
 nonPersistentContextServiceRef$Ref=Thread context propagation reference
 nonPersistentContextServiceRef.desc=The context service is used to manage context propagation to non-persistent timer method threads.
+
+missedPersistentTimerAction=Missed persistent timer action
+missedPersistentTimerAction.desc=Specifies the action to perform when the expiration of an interval or schedule based persistent timer is missed. One or more expirations of a persistent timer are classified as missed if the current expiration is scheduled prior to application server start or the next expiration is scheduled prior to the current time. The default action when failover of persistent timers is enabled is ONCE, otherwise the default action is ALL.
```

Comment:
Change "schedule based" to "Change "schedule-based"
Change "scheduled prior to application server start" to "scheduled before the application server start"
Change "next expiration is scheduled prior to the current time" to "next expiration is scheduled before the current time"

Reply:
done